### PR TITLE
Add form contact

### PR DESCRIPTION
**Pull request by aron-helu**
Adding Form for easier user experience (UI). For the mobile version accept input name and email, and text message to contact the author of the website. 
### cheers :smiley:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -185,6 +186,42 @@ <h1 class="multi-post-header">Multi-Post Stories</h1>
...
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="jeremy@gmail.com" required>
+                </li>
+                <li>
+                    <label for="msg"></label>
+                    <textarea name="user_message" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
+                </li>
+            </ul>
```
Comment (tresorsawasawa) [inline index.html:208]: - [ ] Kindly make sure you used the correct `text-color` on inputs like on the Figma template as some of them have the same **color**  for `text` and `background` and when typing any message, it cannot be visible.
- [ ] Make sure you also used the correct `width` on all form's elements and `margin)` between form's elements and `padding` around text in inputs fields to make your design stick to the Figma template.

File: index.html [hunk 2]
```diff
@@ -185,6 +186,42 @@ <h1 class="multi-post-header">Multi-Post Stories</h1>
...
+        <footer>
+            <!-- <hr class="hr-footer"> -->
+            <ul class="footer-account-links">
+                <li><a href="https://github.com/aron-helu" class="link-footer"><img src="./image/Vector.svg" alt="github-icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/aron-abraham-90a4321b0/" class="link-footer"><img src="./image/Icon-Linkedin.svg" alt="Linkedin-github"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_hand.svg" alt="Peace-hand"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_Twit.svg" alt="Twitter-icon"></a></li>
+                <li><a href="#" class="link-footer"><img src="./image/Vector_M.svg" alt="Medea"></a></li>
+            </ul>
+            <h2>aronmilahela@gmail.com</h2>
```
Comment (tresorsawasawa) [inline index.html:220]: - [ ] Kindly make sure the social icons and the email link are correctly centered like from the Figma template(you can use flexbox to easily center them) and the `bottom` **horizontal** line to match the styling from the Figma. Also, make sure you remove the comments code lines to make your code clean. By fixing these issues your project will stick to the Figma template.
#### Figma design
ff
#### Your design
fs

File: style.css [hunk 3]
```diff
@@ -286,3 +286,114 @@ h5 {
...
+}
+
+.form-paragraph {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  padding: 10%;
+}
```
Comment (tresorsawasawa) [inline style.css:305]: - [ ] Kindly make sure the `form-paragraph` text uses the correct color and spacing(top & bottom) like the one from the Figma template to make your project sticks to the Figma design.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @aron-helu 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aron-helu** [comment]
Fixing required changes by the reviewer asked .  wrap the form inside the footer tag. justify content to center by flexbox. Other content that needs to change.

### cheers :smiley:

**Review by Emmyn5600 — APPROVED** [review]
Hi @aron-helu 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**